app/package.json

```diff
@@ -34,9 +34,11 @@
     "typescript": "^3.6.4",
     "typed-scss-modules": "^0.0.12",
     "source-map-loader": "^0.2.4",
-    "ts-loader": "^6.2.0"
+    "ts-loader": "^6.2.0",
+    "serve": "^11.2.0"
   },
   "scripts": {
+    "serve": "serve -s b",
```

Comment:
I think this should be `serve -s build`

Reply:
yes, accidently deleted `uild` before commit